demisto_sdk/__main__.py

```diff
@@ -815,8 +817,8 @@ def run_playbook(**kwargs):
 @click.option(
     "-v", "--verbose", is_flag=True, help="Verbose output - mainly for debugging purposes")
 @click.option(
-    "--interactive", help="If passed, then for each output field will ask user interactively to enter the "
-                          "description. By default is interactive mode is disabled", is_flag=True)
+    "-int", "--interactive", help="If passed, then for each output field will ask user interactively to enter the "
+                                  "description. By default is interactive mode is disabled", is_flag=True)
```

Comment:
why do we mention the default here and not just set in default value (which is then added by click to the desc)?

Reply:
we can do that...